### ISSUE TITLE
Is it posible to implement some killing function for the org-brain entry? Like org-id-store-link, for org-insert-link.

### ISSUE THREAD

**Issue by techapu** [Kungsgeten/org-brain#163]
Can be this implemented in some way? It would be a fast way to reference an entry for future yanking without leaving the org-mode editing.
I tried to use the org-id-copy function but it doesn't recognize the data later in org-brain when I tried to reference the entry.
Org-store-link and org-insert-link are in my muscle memory of org-mode. It's natural org-mode workflow.

**Comment by Kungsgeten** [comment]
The normal `org-store-link` (which I have bound to `C-c l`) should work fine when used from `org-brain-visualize`. This should create a `brain:` link in the org kill-ring. If I understand correctly, you also want to do this when in `org-mode` and editing an entry?

**Comment by techapu** [comment]
Yes that is right. On your recomendation, I tried that function from `org-brain-visualize` and works as you say, I didn't knew. 
But the way I would benefit from this function working at the org-mode view. Not only killing the headline with the `org-store-link-"brain"` but also yanking the data in the same way, with org-insert-link.

**Comment by Kungsgeten** [comment]
Yes, it should be possible to do from `org-mode` as well I guess. I'll see what I can do.
